### DISCUSSION TITLE
How about a Gdevelop version for Android and iOS touch devices

### DISCUSSION THREAD

**Discussion by JeanBR01** [4ian/GDevelop#3629] [Enhancement ideas (and their technical discussions)]
Mobile devices are evolving a lot and we developers are running out of time, having the Gdevelop tool on Android or IOS would help to continue projects even away from home; Life is busy and I miss Gdevelop in the palm of my hand when ideas come to me and I'm away from home without being able to project them; Please make a mobile version.

**Comment by 4ian** [comment]
Please vote for it on the public roadmap ;) https://trello.com/c/hV7aaiz1/72-port-gdevelop-5-to-android-ios-chromebook
